e2e/preferences.e2e.spec.ts

```diff
@@ -0,0 +1,144 @@
+import os from 'os';
+import path from 'path';
+import { ElectronApplication, BrowserContext, _electron, Page } from 'playwright';
+import { test, expect } from '@playwright/test';
+import { createDefaultSettings, packageLogs, reportAsset } from './utils/TestUtils';
+import { NavPage } from './pages/nav-page';
+import { PreferencesPage } from './pages/preferences';
+
+let page: Page;
+
+/**
+ * Using test.describe.serial make the test execute step by step, as described on each `test()` order
+ * Playwright executes test in parallel by default and it will not work for our app backend loading process.
+ * */
+test.describe.serial('Main App Test', () => {
+  let electronApp: ElectronApplication;
+  let context: BrowserContext;
+  let preferencesWindow: Page;
+
+  test.beforeAll(async() => {
+    createDefaultSettings();
+
+    electronApp = await _electron.launch({
+      args: [
+        path.join(__dirname, '../'),
+        '--disable-gpu',
+        '--whitelisted-ips=',
+        // See src/utils/commandLine.ts before changing the next item as the final option.
+        '--disable-dev-shm-usage',
+        '--no-modal-dialogs',
+      ],
+      env: {
+        ...process.env,
+        RD_LOGS_DIR: reportAsset(__filename, 'log'),
+      },
+    });
+    context = electronApp.context();
+
+    await context.tracing.start({ screenshots: true, snapshots: true });
+    page = await electronApp.firstWindow();
+
+    new NavPage(page).preferencesButton.click();
+
+    await electronApp.waitForEvent('window');
+    await electronApp.waitForEvent('window');
+
+    const windows = electronApp.windows();
+
+    preferencesWindow = windows.find(w => w.url().toLocaleLowerCase().includes('preferences')) || await electronApp.firstWindow();
+  });
+
+  test.afterAll(async() => {
+    await context.tracing.stop({ path: reportAsset(__filename) });
+    await packageLogs(__filename);
+    await electronApp.close();
+  });
+
+  test('should open preferences modal', () => {
+    expect(preferencesWindow).toBeDefined();
+    expect(preferencesWindow?.url().toLowerCase()).toContain('preferences');
+  });
+
+  test('should land application behavior tab', async() => {
+    const preferencesPage = new PreferencesPage(preferencesWindow);
+
+    await expect(preferencesPage.applicationNav).toHaveClass('preferences-nav-item active');
+
+    if (!os.platform().startsWith('win')) {
+      await expect(preferencesPage.behaviorTab).toHaveText('Behavior');
+      await expect(preferencesPage.administrativeAccess).toBeVisible();
+    }
+
+    await expect(preferencesPage.automaticUpdates).toBeVisible();
+    await expect(preferencesPage.statistics).toBeVisible();
+    await expect(preferencesPage.pathManagement).not.toBeVisible();
+  });
+
+  /**
+   * Checking Application Environment tab - macOS & Linux only
+   */
+  if (!os.platform().startsWith('win')) {
```

Comment:
We should have an inverse to check that it exists on Windows.

In Playwright (the E2E scripts; _not_ in the jest unit tests), we can also do:
```ts
test('… description …', async() => {
  test.skip(os.platform() === 'win32', 'Environment tab not available on Windows');
  …
}
```
(See [docs](https://playwright.dev/docs/api/class-test#test-skip-3). And yeah, the existing tests don't do this…)